index.js

```diff
@@ -310,6 +327,10 @@ const ContainerService = {
   },
 
   sortTabs() {
+    sendEvent({
```

Comment:
Assuming we always want to send the tab count this was something I was going to suggest.